README.md

```diff
@@ -60,70 +63,83 @@ The backend will now run at localhost:8000
    - [Sublime](https://github.com/SublimeLinter/SublimeLinter-flake8)
    - [flake8](http://flake8.pycqa.org/en/latest/index.html)
 
-### General setup 
+### General setup
 
-* Create a fork of the repository in your GitHub account, and clone it.
+- Create a fork of the repository in your GitHub account, and clone it.
 
-* Prepare the database:
+- Copy local config files. From inside the dir root, run
+
+```
+cp db_config.sample.py src/config_local/db.py
+cp keys.sample.py src/config_local/keys.py
+cp twitter_config_sample.py src/config_local/twitter.py
+```
 
-    Create a db file in config
+- Prepare the database:
 
-    ```shell
-    touch src/config/db.py
-    ```
+  Update the db file in config
 
-    Add the following:
+  Add the following:
 
-    ```python
-    NAME = 'researchhub'
-    HOST = 'localhost'
-    PORT = 5432
-    USER = 'rh_developer'  # replace as needed
-    PASS = 'not_secure'  # replace as needed
-    ```
+  ```python
+  NAME = 'researchhub' # replace as needed
+  HOST = 'localhost'
+  PORT = 5432
+  USER = 'rh_developer'  # replace as needed
+  PASS = 'not_secure'  # replace as needed
+  ```
 
- * Use [posgres.app](https://postgres.app/) to install Posgres DB. The latest available DB version should be fine.
+- Use [posgres.app](https://postgres.app/) to install Posgres DB. The latest available DB version should be fine.
 
+> Good UI tool for interacting with PostgreSQ: [Postico](https://eggerapps.at/postico2/)
 
-  > Good UI tool for interacting with PostgreSQ: [Postico](https://eggerapps.at/postico2/)
+- The project virtual environment is managed using [Poetry](https://python-poetry.org/docs/).
 
-* The project virtual environment is managed using [Poetry](https://python-poetry.org/docs/).
   ```shell
   pip3 install poetry
   ```
 
-* Go to the [`src`](src) directory and run the following commands in order to activate the virtual environment:
-    ```shell
-    cd src
+- Make sure to set your local shell python version to 3.9 as Poetry uses the python version of the current shell as the default python version. e.g. using pyenv:
```

Comment:
This is new, possibly not correct, but based v3.9 on @manveerxyz's set up and it worked for me so 🤷‍♂️